webapp/Connector/src/view/Grid.js

```diff
@@ -579,7 +618,7 @@ Ext.define('Connector.view.Grid', {
         var model = this.getModel(),
             modelMap = {},
             columns,
-            models = model.get('metadata').columnModel,
+            models = model.getActiveSheetMetadata().columnModel,
             applyChecker = false,
             queryService = Connector.getService('Query');
```

Comment:
I realize this wasn't modified but this could be updated to use `Connector.getQueryService()`.